### PR TITLE
update setup.tra & prompts.tra

### PR DESCRIPTION
**Pull request by Lzw104522773**
Modify several NPC name translations to achieve consistency. Modify prompts.tra encoding to UTF-8 to avoid garbled prompts during installation with Weidu version 249.

### CODE REVIEW

File: DSotSC/language/schinese/prompts.tra [hunk 1]
```diff
@@ -1,77 +1,77 @@
 // WeiDU prompts
-@-1000 = ~�ڰ��������~
```
Comment (Roberciiik) [inline DSotSC/language/schinese/prompts.tra:2]: Please add prompts.tra file to charsetNoConvert variable list.

File: DSotSC/DSotSC.tp2 [hunk 2]
```diff
@@ -40,8 +40,8 @@ ASK_EVERY_COMPONENT
 /////                                                  \\\\\
 
 ALWAYS
-<<<<<<<< .../blank.txt
```
Comment (Roberciiik) [inline DSotSC/DSotSC.tp2:43]: Please revert this change

**Comment by Roberciiik** [comment]
@Lzw104522773  Please resolve merge conflicts. Seems that you missed my previous message.

File: DSotSC/DSotSC.tp2 [hunk 3]
```diff
@@ -78,6 +78,31 @@ ALWAYS
 
 	INCLUDE ~DSotSC/lib/macros.tpa~
 
+
+	ACTION_IF ENGINE_IS ~bgee bg2ee eet~ BEGIN
```
Comment (Roberciiik) [inline DSotSC/DSotSC.tp2:82]: @Lzw104522773 After the merge, this section is duplicated. Probably because of unresolved conflicts. Please compare and add only proper change to your commit.

**Comment by Lzw104522773** [comment]
Refork and resubmit.

**Comment by Lzw104522773** [comment]
refork and resubmit.
